### PR TITLE
Refactor code and update README

### PR DESCRIPTION
**Pull request by repeatedly**

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

File: lib/fluent/plugin/in_systemd.rb [hunk 1]
```diff
@@ -13,68 +13,65 @@ class SystemdInput < Input
     config_param :strip_underscores, :bool, default: false
     config_param :tag, :string
 
-    attr_reader :tag
+    attr_reader :tag, :journal, :running, :pos_writer
```
Comment (errm) [inline lib/fluent/plugin/in_systemd.rb]: These `attr_readers` are not needed if we are using instance variables to access the values...

File: lib/fluent/plugin/systemd/pos_writer.rb [hunk 2]
```diff
@@ -8,50 +8,48 @@ def initialize(pos_file)
         setup
       end
 
-      attr_reader :cursor, :path
+      attr_reader :cursor, :path, :lock, :thread, :running
```
Comment (errm) [inline lib/fluent/plugin/systemd/pos_writer.rb]: same here with unused `attr_readers`

**Review by errm — CHANGES_REQUESTED** [review]
Basically looks good to me.

Could you remove the unused `attr_reader` declarations as there is no need to expose everything outside these classes...

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by repeatedly** [comment]
Applied reviews

**Review by errm — APPROVED** [review]

**Pull request closed without merging**